test/verify/check-machines-create

```diff
@@ -755,7 +755,7 @@ class TestMachinesCreate(VirtualMachinesCase):
             self.browser.wait_text("#vm-{}-disks-sda-device".format(self.name), "cdrom")
             self.goToMainPage()
 
-            virt_install_cmd = "ps aux | grep 'virt\-install\ \-\-connect'"
+            virt_install_cmd = "ps aux | grep '[v]irt-install --connect'"
```

Comment:
That `[v]` needs an explanation. How is that any different from just `v`?

Reply:
I thought this would have fallen under the "usual tricks" category: this is to avoid `grep` finding itself in the output of `ps`.  The `[v]` approach is even the more idiomatic one...

Reply:
I'll add a comment